crates/networking/src/networking.rs

```diff
@@ -214,12 +227,26 @@ impl Drop for MultiplexedReceiver {
 // we need to make a key that is unique for each (send->dest) pair and stream.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize)]
 struct CompoundStreamKey {
-    stream_id: StreamKey,
-    send_user_id: UserID,
-    recv_user_id: Option<UserID>,
+    stream: StreamKey,
```

Comment:
```suggestion
    stream_key: StreamKey,
```
Seems like the name can be more clear.